app/models/container.rb

```diff
@@ -104,6 +108,20 @@ def update_bootstrappers(bootstrappers)
     end
   end
 
+  def update_source(new_source)
```

Comment:
the name of method is update, but I don't see we do update in the body of the method

Reply:
cc @galuhest.

Reply:
there was a mistake there, i've fixed it mas